package.json

```diff
@@ -36,7 +36,7 @@
     "js-base64": "^2.5.0"
   },
   "devDependencies": {
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
```

Comment:
There are problems with the changes, if upgrade gulp to v4 from v3.

Reply:
We should think twice about upgrading gulp, close this PR momentarily.

Reply:
> We should think twice about upgrading gulp, close this PR momentarily.

👌

Reply:
> We should think twice about upgrading gulp, close this PR momentarily.

Approve of your comment.